ehrapy/tools/_sa.py

```diff
@@ -262,3 +263,33 @@ def anova_glm(result_1: GLMResultsWrapper, result_2: GLMResultsWrapper, formula_
     }
     dataframe = pd.DataFrame(data=table)
     return dataframe
+
+
+def cox_ph(adata: AnnData, duration_col: str, event_col: str, entry_col: str = None) -> KaplanMeierFitter:
+    """Fit the Cox’s proportional hazard for the survival function.
```

Comment:
Could you maybe add a few details on the model below before referring to lifelines, please? If we don't have that for Kaplan Meier either, it'd be good to add this to this PR